docs/recipes/delete-files-folder.md

```diff
@@ -70,13 +70,20 @@ var del = require('del');
 var vinylPaths = require('vinyl-paths');
 
 gulp.task('clean:tmp', function () {
+  var vp = vinylPaths();
   return gulp.src('tmp/*')
+    .pipe(vp)
     .pipe(stripDebug())
     .pipe(gulp.dest('dist'))
-    .pipe(vinylPaths(del));
```

Comment:
after that, it will delete the file pipe to the `dist`, it won't delete the files in the `tmp`